packages/block-library/src/tag-cloud/index.php

```diff
@@ -29,8 +29,13 @@ function render_block_core_tag_cloud( $attributes ) {
 	);
 	$tag_cloud = wp_tag_cloud( $args );
 
-	if ( ! $tag_cloud ) {
-		$tag_cloud = __( 'There&#8217;s no content to show here yet.' );
+	if ( empty( $tag_cloud ) ) {
+		// Display placeholder content when there are no tags only in editor.
+		if ( ( defined( 'REST_REQUEST' ) && REST_REQUEST ) || is_admin() ) {
+			$tag_cloud = __( 'There&#8217;s no content to show here yet.' );
+		} else {
```

Comment:
WP introduced the `wp_is_serving_rest_request` method, which we could use here. The `is_admin` check shouldn't be required.

Reply:
Thank your for a review. I have addressed your feedback.